test/torchaudio_unittest/prototype/functional/torchscript_consistency_test_impl.py

```diff
@@ -85,6 +85,18 @@ def test_sinc_ir(self):
     def test_speed(self):
         leading_dims = (3, 2)
         T = 200
-        waveform = torch.rand(*leading_dims, T, dtype=self.dtype, device=self.device, requires_grad=True)
+        waveform = torch.rand(*leading_dims, T, dtype=self.dtype, device=self.device)
```

Comment:
was this meant to be modified?

Reply:
yeah i realized it wasn't necessary, but i can deal with it in a separate pr